src/library_wasi.js

```diff
@@ -138,6 +138,7 @@ var WasiLibrary = {
   clock_time_get__nothrow: true,
   clock_time_get__sig: 'iiiii',
   clock_time_get__deps: ['emscripten_get_now', 'emscripten_get_now_is_monotonic', '$checkWasiClock'],
+  clock_time_get__docs: {{{ documentI64Param('precision') }}},
   clock_time_get: function(clk_id, {{{ defineI64Param('precision') }}}, ptime) {
```

Comment:
Could the docs be inline inside `defineI64Param`? I'm hoping closure will parse a comment right before the parameter and not just a comment before the entire function...

Reply:
I tried that for quite a long time .. and I couldn't make it work at all.   I caused the library file to fail to parse.  I will try one more time.